server/controllers/reports_proposed/data/balance_sheet.js

```diff
@@ -1,5 +1,11 @@
-// reports_proposed/data/balance_sheet.js
```

Comment:
What you are asking for is not a balance sheet, so it doesn't apply to this PR.  This PR fixes/creates a balance sheet which filters only on balance accounts.